src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -293,16 +353,31 @@ private string ClearHexString(string hexString)
         }
 
         /// <summary>
-        /// Converts an address to Base64 byte array
+        /// Address to ScriptHash (blittleig-endian)
+        /// input:  NejD7DJWzD48ZG4gXKDVZt3QLf1fpNe1PF
+        /// output: ce616f7f74617e0fc4b805583af2602a238df63f
         /// </summary>
-        /// <param name="address">
-        /// String that represents the address to be converted
-        /// </param>
-        /// <returns>
-        /// Returns null when the string does not represent an address or when it is 
-        /// not possible to parse the address to Base64 value; otherwise returns
-        /// the string that represents the converted Base64 value.
-        /// </returns>
+        [ParseFunction("Address to ScriptHash (little-endian)")]
+        private string? AddressToScripthashLE(string address)
+        {
+            try
+            {
+                var bigEndScript = address.ToScriptHash(NeoSystem.Settings.AddressVersion);
+
+                return bigEndScript.ToArray().ToHexString();
```

Comment:
```suggestion
                return address.ToScriptHash(NeoSystem.Settings.AddressVersion);
```

Reply:
This will output a result of type UInt160, not a string.